init.lua

```diff
@@ -7,7 +7,23 @@ if vim.fn.empty(vim.fn.glob(install_path)) > 0 then
   vim.cmd [[packadd packer.nvim]]
 end
 
-require('packer').startup(function(use)
+-- Use a protected call so we don't error out on first use
+local status_ok, packer = pcall(require, "packer")
+if not status_ok then
+  return
+end
+local status_util_ok, util = pcall(require, "packer.util")
```

Comment:
Really like the `pcall` addition! Had an error when I first ran it because of the issue you're solving here 🙌 

What do you think about using `packer_util` instead of just `util`? I think it would be more semantic that way.

Reply:
Is is a shameless copy from https://github.com/LunarVim/Neovim-from-scratch/blob/2c7a64973602e8d9d18f6de75022014e96fa576a/lua/user/plugins.lua#L26-L30 =D

About the `packer_util`, done in 7881ea08439227fbda12d07c1fb3e6b8dbace53e

Reply:
Great 🙌